src/components/projekti/tiedottaminen/OmistajienMuokkausLomake.tsx

```diff
@@ -477,9 +444,20 @@ const MuokkausDialogContent: VFC<{
           }
           if (apiData) {
             try {
-              await api.tallennaKiinteistonOmistajat(apiData);
-              useFormReturn.reset(data);
-              close();
+              const ids = await api.tallennaKiinteistonOmistajat(apiData);
+              const newData: KiinteistonOmistajatFormFields = {
+                oid: data.oid,
+                suomifiOmistajat: data.suomifiOmistajat.filter((m) => !apiData?.poistettavatOmistajat.includes(m.id ?? "")),
+                muutOmistajat: data.muutOmistajat.filter((m) => !apiData?.poistettavatOmistajat.includes(m.id ?? "")),
+                lisatytOmistajat: data.lisatytOmistajat.filter((m) => !apiData?.poistettavatOmistajat.includes(m.id ?? "")),
+              };
+              ids.splice(0, newData.muutOmistajat.length);
```

Comment:
Sus. BE:ssä on ilmeisesti tehty niin, että muutOmistajat on id:ssä ekana, jolloin näin voi tehdä, mutta ehkä vähän riskaabeli tapa.